packages/phoenix-otel/src/phoenix/otel/otel.py

```diff
@@ -227,7 +227,7 @@ def __init__(self, *args: Any, **kwargs: Any):
         if bound_args.arguments.get("endpoint") is None:
             _, endpoint = _normalized_endpoint(None)
             bound_args.arguments["endpoint"] = endpoint
-        super().__init__(*args, **kwargs)
+        super().__init__(**bound_args.arguments)
```

Comment:
not sure I know the difference. lol

Reply:
This didnt pipe in the headers if a user set them for our GRPCExporter :(